### PR TITLE
JDF-465 Added a quickstart with Bootstrap and AngularJS

### PR DESCRIPTION
**Pull request by VineetReynolds**
This is a port of the kitchensink-angularjs quickstart that uses Bootstrap 3 for styling.

### CODE REVIEW

File: kitchensink-angularjs-bootstrap/README.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+kitchensink-angularjs-bootstrap: Shows how to use AngularJS and Twitter Bootstrap with JAX-RS and Java EE on JBoss
+========================
+Author: Pete Muir
```
Comment (pmuir) [inline kitchensink-angularjs-bootstrap/README.md]: You might want to change this!

Comment (VineetReynolds) [inline kitchensink-angularjs-bootstrap/README.md]: If you say so :) I didn't do much except change the original QS to use
Bootstrap.

**Comment by pmuir** [comment]
I think you need to add this to the root pom.xml

**Comment by pmuir** [comment]
I haven't done a detailed review, but I don't see too many issues. My main one is that we need to remove or tidy up screen.css, to remove unused styles (e.g. simpletablestyle has been replaced by bootstrap).

**Pull request closed without merging**